app/(white_navbar)/explore/RetroPGF5/page.tsx

```diff
@@ -0,0 +1,54 @@
+import { BadgeholderMetrics } from '@/app/(white_navbar)/explore/RetroPGF4/RetroType4'
+import Cpage from './Cpage'
+import BreadcrumbExplore from './_component/BreadcrumbExplore'
+import fs from 'fs'
+import path from 'path'
+import { RetroPGF5Project } from './type'
+
+async function getJsonRound5(): Promise<RetroPGF5Project[]> {
+  const directoryPath = path.join(
+    process.cwd(),
+    'public/static/rpgf5/dummy_rpgf5.json'
+  )
+  const fileContents = await fs.promises.readFile(directoryPath, 'utf8')
+  const jsonData: RetroPGF5Project[] = JSON.parse(fileContents)
+  return jsonData
+}
+
+async function getJsonBadgeholderMetric(): Promise<BadgeholderMetrics[]> {
+  const directoryPath = path.join(
+    process.cwd(),
+    'public/static/rpgf4/rpgf4_badgeholder_metrics.json'
```

Comment:
Don't forget the change the path to round5 if want to use the data of round5

Reply:
This going to be change later when the round concludes. Thank for notice 👍